zipline/data/ffc/loaders/_us_equity_pricing.pyx

```diff
@@ -30,13 +30,13 @@ from zipline.data.adjustment import Float64Multiply
 
 SID_QUERY_TEMPLATE = """
 SELECT DISTINCT sid FROM {0}
-WHERE effective_date >= {1} AND effective_date <= {2}
+WHERE effective_date >= (?) AND effective_date <= (?)
```

Comment:
I think this would be easier to follow if we did the interpolation on these templates by name.